server/sonar-server/src/main/java/org/sonar/server/qualitygate/ws/GetByProjectAction.java

```diff
@@ -90,8 +93,13 @@ public void handle(Request request, Response response) throws Exception {
         !userSession.hasComponentPermission(UserRole.ADMIN, project)) {
         throw insufficientPrivilegesException();
       }
+      Optional<OrganizationDto> organization = dbClient.organizationDao().selectByUuid(dbSession, project.getOrganizationUuid());
 
-      Optional<QualityGateData> data = qualityGateFinder.getQualityGate(dbSession, project.getId());
+      if (!organization.isPresent()) {
+        throw new NotFoundException(format("Organization [%s] is absent", project.getOrganizationUuid()));
+      }
```

Comment:
Please use QualityGateWsSupport#getOrganization (QualityGateWsSupport is already used in this class, you only need to rebase on the common branch)